### PR TITLE
docs: added kind deployment article

### PR DESCRIPTION
**Pull request by mbshields**
**What type of PR is this?**

documentation

**Which issue does this PR fix**:

none

**What does this PR do / Why do we need it**:

Adds an article for deploying a kind cluster with a zot registry

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/articles/kind-deploy.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+
+```shell
+{% raw %}
+# create registry container unless it already exists
+reg_name='kind-registry'
+reg_port='5001'
+if [ "$(docker inspect -f '{{.State.Running}}' "${reg_name}" 2>/dev/null || true)" != 'true' ]; then
+  docker run \
+    -d --restart=always -p "127.0.0.1:${reg_port}:5000" --name "${reg_name}" \
+    ghcr.io/project-zot/zot-minimal-linux-amd64:latest
```
Comment (rchincha) [inline docs/articles/kind-deploy.md:58]: Add a note that `ghcr.io/project-zot/zot-minimal-linux-amd64:latest` with whatever zot version one wants

Comment (mbshields) [reply docs/articles/kind-deploy.md:55]: Added suggestion in Step 2 to replace 'latest' with release number if desired.

File: docs/articles/kind-deploy.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
+# Using kind for Deployment Testing
```
Comment (rchincha) [inline docs/articles/kind-deploy.md:1]: `kind` so it stands out

Comment (rchincha) [reply docs/articles/kind-deploy.md:1]: Same below everywhere

Comment (mbshields) [reply docs/articles/kind-deploy.md:1]: Do you mean to highlight the word 'kind' such as bold or code notation?  That can be done throughout, although not in the title itself.

**Review by rchincha — APPROVED** [review]

**Pull request merged**